src/components/Nav/index.tsx

```diff
@@ -362,13 +362,31 @@ const Nav: React.FC<IProps> = ({ path }) => {
       to: "/developers/local-environment/",
     },
   ]
-
+  useEffect(() => {
+    // Event listener for when user clicks on .DocSearch-Container div, to toggle isSearchOpen to false
+    const closeSearchListener = (e) => {
+      if (
+        e.target &&
+        (e.target.className === "DocSearch-Container" ||
+          e.target.className === "DocSearch-Cancel")
+      ) {
+        setIsSearchOpen(false)
+        setIsMenuOpen(false)
+      }
+    }
+    document.addEventListener("click", closeSearchListener)
+    // remove event listener on dismount
+    return () => {
+      document.removeEventListener("click", closeSearchListener)
+    }
```

Comment:
Need to debug this still... having trouble getting this to work appropriately on mobile.

Clicking the top search icon on mobile opens the modal, but then clicking/tapping outside of the modal (along the 1rem padding) will close the search but leaves the backdrop from the menu open.

![image](https://user-images.githubusercontent.com/54227730/220829172-084dd2bf-feb4-4aaf-8eea-04b2260a9abe.png)

Tapping/clicking this then removes it, but if you click the menu icon a few times it will toggle the menu without removing this backdrop.